h_transport_materials/references.bib

```diff
@@ -2606,3 +2606,18 @@ @article{piper_diffusion_2004
     url = {https://doi.org/10.1063/1.1708243},
     eprint = {https://pubs.aip.org/aip/jap/article-pdf/37/2/715/7937414/715\_1\_online.pdf},
 }
+
+@article{Guthrie1974,
```

Comment:
```suggestion
@article{guthrie_permeation_1974,
```

The tag of the bibtex entry has to be the same as the one given in the property (here `guthrie_permeation_1974`)